File: app/src/main/java/com/tolstoy/zurichat/ui/otp/OTP_Page.java
package com.tolstoy.zurichat.ui.otp;

import static com.tolstoy.zurichat.util.UtilitiesKt.setUpApplicationTheme;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.tolstoy.zurichat.R;

public class OTP_Page extends AppCompatActivity {
    private Button otpVerifyCodeBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_otp_page);

        // This setups application theme to value stored in sharedPref
        setUpApplicationTheme(this);

        otpVerifyCodeBtn = findViewById(R.id.otpVerifyCodeBtn);
        otpVerifyCodeBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), VerifyOTP.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/tolstoy/zurichat/ui/otp/OTP_Page.kt
package com.tolstoy.zurichat.ui.otp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.tolstoy.zurichat.R
import com.tolstoy.zurichat.databinding.ActivityOtpPageBinding
import com.tolstoy.zurichat.ui.activities.HomePageActivity
import com.tolstoy.zurichat.ui.activities.MainActivity

class   OTP_Page : AppCompatActivity() {

    private val binding: ActivityOtpPageBinding by lazy {
        ActivityOtpPageBinding.inflate(layoutInflater)
    }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
        val verifyCodeBtn: Button = findViewById(R.id.otpVerifyCodeBtn)

        verifyCodeBtn.setOnClickListener {
            val int = Intent(this, HomePageActivity::class.java)
            startActivity(int)
        }

    }
}